Construct/Encouter/CombatantRow.swift

```diff
@@ -121,8 +124,10 @@ func ShieldIcon(ac: Int) -> some View {
     ZStack {
         Image(systemName: "shield")
             .font(Font.title.weight(.light))
+            .accessibility(hidden: true)
         Text("\(ac)")
             .font(.caption)
+            .accessibility(label: Text("\(ac) armor class"))
     }
```

Comment:
I think it would be cleaner to add the `accessibilityElement()` modifier to the wrapper ZStack and set the label on the whole. That would remove the need to explicitly hide the Image. What do you think?

Reply:
Fixed this in 0118584.